patches/minecraft/net/minecraft/server/dedicated/ServerHangWatchdog.java.patch

```diff
@@ -17,6 +17,14 @@
              {
                  field_180251_a.fatal("A single server tick took {} seconds (should be max {})", String.format("%.2f", (float)k / 1000.0F), String.format("%.2f", 0.05F));
```

Comment:
Couldn't you just copy the message from this string (possibly enhanced slightly) and add it to the Error? Would give you information on elapsed time as well.

Reply:
I think asie's idea is good, please copy this message.